README.md

```diff
@@ -157,7 +157,23 @@ The Converged repository must contain release branches for all prior stable line
 
 ### Long Term Support WG
 
-All Interim and Post-Convergence Releases other than current automatically shift into the responsibility of the Long Term Support working group. The specific approach for determining the LTS strategy for specific releases is entirely in the LTS WG’s hands. 
+All Interim and Post-Convergence Releases other than current automatically shift into the responsibility of the Long Term Support working group. The specific approach for determining the LTS strategy for specific releases is entirely in the LTS WG’s hands.
+
+Every Major Release creates an opportunity to establish a new Long Term Support tag. For instance, suppose the master branch has a Major version bump to 2.0. The LTS WG can decide that there will be a LTS release in the 2.x stream. Any Minor release can be declared as an "LTS Candidate". Sufficient time should then be given to allow the branch to be reviewed and tested. Numerous Patch versions may occur during this time. Once the LTS WG is satisfied that the LTS Candidate is stable, it would cut the new LTS tag. It's possible that Minor version increases will occur in master while testing and reviewing the LTS Candidate.
+
+Example:
+* Major version increased to 2.0.0
+* LTS WG declares that 2.0.0 is an LTS Candidate
+* Zero or more Patches on 2.0.x occur such that the current version is 2.0.10
+* LTS WG declares that 2.0.10 is the LTS Version
+
+Example:
+* Major version increased to 2.5.0
+* LTS WG declares that 2.5.0 is an LTS Candidate
+* Zero or more Patches on 2.5.x occur such that the current version is 2.5.6
+* A PR occurs that forces a Minor version increase to 2.6.0,
+* The LTS WG decides to include the Minor increase in the LTS, 2.6.0 becomes the new LTS Candidate
+* LTS WG declares that 2.6.1 is the LTS Version
```

Comment:
Probably needs to be specified here and in the above description that this only works for using the latest minor if we are able to still bump minors.